_config.yml

```diff
@@ -44,25 +44,25 @@ organizerLink: "http://alumniei.pt/"
 navigationLinks:
  #- {permalink: "/", text: "Home"}
 bottomNavigationLinks:
- - {link: "https://forms.gle/as4vi5bJuai7XZnr5", text: "Share your CV"}
+ - {link: "https://forms.gle/SJTv7bZHUJNUULd57", text: "Share your CV"}
```

Comment:
Couldn't you make the form url a variable?
See [this](https://stackoverflow.com/questions/41620674/use-placeholders-in-yaml)

Reply:
I mean, declare the variable at the beginning of the `_config.yml` and then use the variable where needed.